### ISSUE TITLE
changes to sge_resource_param in config file don't make it through to backend.conf

### ISSUE THREAD

**Issue by welchwilmerck** [ENCODE-DCC/caper#177]
Release 2.2.0
After `caper init sge`, I add a custom parameter to sge_resource_param.  Upon `caper run`, the generated backend.conf has the hard-coded default sge_resource_param, not the custom one.

I've isolated the problem to the init of the CaperRunner object in cli.py https://github.com/ENCODE-DCC/caper/blob/c10d05b62a034a060cc142cdc3e8e103c38518ad/caper/cli.py#L238-L248 and to each of the manager `merge_dict` in caper_backend_conf.py.

The new `resource_param` feature was added only for slurm, but none of the other managers.  If I `caper init slurm` and add a custom parameter to slurm_resource_param, it does override the default in the init of CaperRunner and I see the change in backend.conf

```
$ grep -n resource_param= cli.py
240:        slurm_resource_param=getattr(args, 'slurm_resource_param', None),

$
```

Have to add the manager `resource_param` to each of the manager `merge_dict` in caper_backend_conf.py like is done for slurm.

```
$ grep -nE '(merge_dict\(|_param=.*param)' caper_backend_conf.py
194:        merge_dict(
204:        merge_dict(
224:        merge_dict(
234:        merge_dict(
243:                slurm_extra_param=slurm_extra_param,
244:                slurm_resource_param=slurm_resource_param,
248:        merge_dict(
257:                sge_extra_param=sge_extra_param,
261:        merge_dict(
269:                pbs_extra_param=pbs_extra_param,
273:        merge_dict(
281:                lsf_extra_param=lsf_extra_param,
298:            merge_dict(
313:            merge_dict(
```

**Comment by leepc12** [comment]
Thanks for reporting. Fixed in the above PR. I will make a new release soon.

**Comment by leepc12** [comment]
Fixed in v2.2.2. Please re-open if the problem persists.
